service/src/main/java/com/softserve/borda/services/impl/BoardListServiceImpl.java

```diff
@@ -38,14 +36,14 @@ public BoardList deleteTicketFromBoardList(BoardList boardList, Ticket ticket) {
             ticketService.deleteTicketById(ticket.getId());
             return boardListRepository.save(boardList);
         } catch (Exception e) {
-            throw new CustomFailedToDeleteEntityException(e.getMessage());
+            throw new com.softserve.borda.exceptions.CustomFailedToDeleteEntityException(e.getMessage());
         }
     }
 
     @Override
     public BoardList getBoardListById(Long id) {
         return boardListRepository.findById(id)
-                .orElseThrow(() -> new CustomEntityNotFoundException(BoardList.class));
+                .orElseThrow(() -> new com.softserve.borda.exceptions.CustomEntityNotFoundException(BoardList.class));
```

Comment:
I would recommend using of ExceptionHandler for CustomEntityNotFoundException (https://www.baeldung.com/exception-handling-for-rest-with-spring)